src/renderers/webgl/WebGLState.js

```diff
@@ -13,7 +13,7 @@ function WebGLState( gl, extensions, paramThreeToGL ) {
 
 		var color = new Vector4();
 		var currentColorMask = null;
-		var currentColorClear = new Vector4();
+		var currentColorClear = new Vector4( 0, 0, 0, 0 );
```

Comment:
This is a gotcha, IMO.

I think it would be better to default `Vector4` to all zeros. We have a separate `Quaternion` class for representing rotations, for which setting the w-component to 1 makes sense.

Also, a `Vector4` instance can represent something other than a homogeneous coordinate (another example for which setting `w` to 1 makes sense.)